### ISSUE TITLE
pcolormesh bug: edgecolor ignored

### ISSUE THREAD

**Issue by bblay** [matplotlib/matplotlib#1302]
I believe the following code should produce grey edges but they have colour.

```
import numpy as np
import matplotlib.pyplot as plt

plt.pcolormesh(np.arange(12).reshape(4, 3), edgecolor='#888888', facecolor='none')
plt.show()
```

**Comment by doylejg** [comment]
Just confirming this bug in 1.4.3, has this been fixed in 1.5.0?

**Comment by efiring** [comment]
Both pcolormesh and pcolor use colormapping based on the input array for the edges when the `facecolor='none'`, regardless of the setting of `edgecolors`.  I note that 'facecolor' is not in the docstrings; it is simply being passed on the Collection.  It sounds like the desired behavior would be for the `facecolor='none'` case to be trapped and used to defeat the colormapping of edges when `edgecolors` is also specified.
